rotkehlchen/tests/unit/accounting/test_misc.py

```diff
@@ -180,3 +185,42 @@ def test_fees_in_received_asset(accountant, google_service):
         AccountingEventType.LEDGER_ACTION: PNL(taxable=FVal('178.615'), free=ZERO),
     })
     check_pnls_and_csv(accountant, expected_pnls, google_service)
+
+
+@pytest.mark.parametrize('should_mock_price_queries', [False])
```

Comment:
Why not mock price queries? You should mock the historical queries to make sure the test is repeatable.

Reply:
I've mocked the result from xrates to verify that the logic is indeed correct by patching the price historian